<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="InetBankingV1" duration-ms="80537" started-at="2021-04-03T12:04:14Z" finished-at="2021-04-03T12:05:35Z">
    <groups>
    </groups>
    <test name="BankingApp Test" duration-ms="80537" started-at="2021-04-03T12:04:14Z" finished-at="2021-04-03T12:05:35Z">
      <class name="com.inetbanking.TestCases.TC_LoginTest_001">
        <test-method status="PASS" signature="Setup(java.lang.String)[pri:0, instance:com.inetbanking.TestCases.TC_LoginTest_001@511baa65]" name="Setup" is-config="true" duration-ms="30592" started-at="2021-04-03T12:04:14Z" finished-at="2021-04-03T12:04:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="loginTest()[pri:0, instance:com.inetbanking.TestCases.TC_LoginTest_001@511baa65]" name="loginTest" duration-ms="5717" started-at="2021-04-03T12:04:45Z" finished-at="2021-04-03T12:04:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.inetbanking.TestCases.TC_LoginTest_001@511baa65]" name="tearDown" is-config="true" duration-ms="3487" started-at="2021-04-03T12:04:51Z" finished-at="2021-04-03T12:04:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.inetbanking.TestCases.TC_LoginTest_001 -->
      <class name="com.inetbanking.TestCases.TC_AddCustomerTest_003">
        <test-method status="PASS" signature="Setup(java.lang.String)[pri:0, instance:com.inetbanking.TestCases.TC_AddCustomerTest_003@340f438e]" name="Setup" is-config="true" duration-ms="16470" started-at="2021-04-03T12:04:54Z" finished-at="2021-04-03T12:05:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="addNewCustomer()[pri:0, instance:com.inetbanking.TestCases.TC_AddCustomerTest_003@340f438e]" name="addNewCustomer" duration-ms="22426" started-at="2021-04-03T12:05:10Z" finished-at="2021-04-03T12:05:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.inetbanking.TestCases.TC_AddCustomerTest_003.addNewCustomer(TC_AddCustomerTest_003.java:65)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.inetbanking.TestCases.TC_AddCustomerTest_003@340f438e]" name="tearDown" is-config="true" duration-ms="1700" started-at="2021-04-03T12:05:33Z" finished-at="2021-04-03T12:05:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.inetbanking.TestCases.TC_AddCustomerTest_003 -->
    </test> <!-- BankingApp Test -->
  </suite> <!-- InetBankingV1 -->
</testng-results>
